contrib/examples/cluster.yaml

```diff
@@ -4,14 +4,47 @@ metadata:
   name: cluster-create-template
 objects:
 
+- apiVersion: clusteroperator.openshift.io/v1alpha1
+  kind: ClusterVersion
+  metadata:
+    name: ocp-v3-9
+    namespace: cluster-operator
+  spec:
+    imageFormat: "openshift/origin-${component}:v3.9.0-alpha.1"
+    vmImages:
+      awsVMImages:
+        amiByRegion:
```

Comment:
This is outside the scope of this PR, but `amiByRegion` should not be a map. See https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#lists-of-named-subobjects-preferred-over-maps.

Reply:
Oh good info, ok adding an issue for that one then.